packages/myst-cli/src/build/utils/collectExportOptions.ts

```diff
@@ -301,6 +306,18 @@ export async function collectExportOptions(
           )),
         );
       }
+      if (formats.includes(ExportFormats.typst)) {
+        fileExportOptionsList.push(
+          ...(await collectTexExportOptions(
```

Comment:
Probably need to reframe this function as something like `collectThingsThatMayOrMayNotBeZippedExportOptions`